curriculum/challenges/english/08-coding-interview-prep/rosetta-code/look-and-say-sequence.md

```diff
@@ -0,0 +1,75 @@
+---
+id: 5e6dd14797f5ce267c2f19d0
+title: Look-and-say sequence
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+The  <a href="https://en.wikipedia.org/wiki/Look and say sequence">Look and say sequence</a>  is a recursively defined sequence of numbers studied most notably by  <a href="https://en.wikipedia.org/wiki/John Horton Conway">John Conway</a>.
+Sequence Definition
+<ul><li>Take a decimal number</li>
+<li><span>Look</span> at the number, visually grouping consecutive runs of the same digit.</li>
+<li><span>Say</span> the number, from left to right, group by group; as how many of that digit there are - followed by the digit grouped.</li></ul><span> This becomes the next number of the sequence.</span>
+<span>An example:</span>
+<ul><li>Starting with the number 1,  you have <span>one</span> 1 which produces 11</li>
+<li>Starting with 11,  you have <span>two</span> 1's.  I.E.:  21</li>
+<li>Starting with 21,  you have <span>one</span> 2, then <span>one</span> 1.  I.E.:  (12)(11) which becomes 1211</li>
+<li>Starting with 1211,  you have <span>one</span> 1, <span>one</span> 2, then <span>two</span> 1's.  I.E.:  (11)(12)(21) which becomes 111221</li></ul>
+</section>
+
+## Instructions
+<section id='instructions'>
+Write a function that takes a string as parameter. This string has the number that should be processed as mentioned above. The function should return the resultant string.
```

Comment:
```suggestion
Write a function that takes a string as parameter. This string has the number that should be processed as mentioned above. The function should return the resultant string.
```
```suggestion
Write a function that takes accepts a string as a parameter, processes it and returns the resultant string.
```

Reply:
Maybe:

Write a function that accepts a string as a parameter, processes it, and returns the resultant string.

Reply:
@scissorsneedfoodtoo I think the suggested one is good to go. @thecodingaviator what do you think?